app/frontend/components/module_assistant/assistant_steps_add.js

```diff
@@ -0,0 +1,251 @@
+/* This file is part of Ezra Bible App.
+
+   Copyright (C) 2019 - 2021 Ezra Bible App Development Team <contact@ezrabibleapp.net>
+
+   Ezra Bible App is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 2 of the License, or
+   (at your option) any later version.
+
+   Ezra Bible App is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with Ezra Bible App. See the file LICENSE.
+   If not, see <http://www.gnu.org/licenses/>. */
+
+
+const { html } = require('../../helpers/ezra_helper.js');
+const assistantController = require('./assistant_controller.js');
+const assistantHelper = require('./assistant_helper.js');
+require('./step_update_repositories.js');
+require('./step_languages.js');
+require('./step_repositories.js');
+require('./step_modules.js');
+require('./step_install.js');
+require('./unlock_dialog.js');
+
+const template = html`
+<style>
+  #module-settings-assistant-add-wrapper {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+  #module-settings-assistant-add {
+    height: 100%;
+  }
+  #module-settings-assistant-add .content {
+    min-height: calc(100% - 4em);
+  }
+  #module-assistant-add-info {
+    padding: 0 2.5% 1em 0.5em;
+  }
+</style>
+
+<div id="module-settings-assistant-add-wrapper" style="display: none;">
+  <div id="module-assistant-add-info">
+    <step-update-repositories></step-update-repositories>
+  </div>
+
+  <div id="module-settings-assistant-add"></div>
+</div>
+
+<unlock-dialog></unlock-dialog>
+`;
+
+const templateAddSteps = html`
+  <h3 i18n="module-assistant.languages"></h3>
+  <section id="module-languages" class="module-assistant-step scrollable">
+    <!-- <step-languages></step-languages> -->
+  </section>
+
+  <h3 i18n="module-assistant.repositories"></h3>
+  <section id="module-repositories" class="module-assistant-step scrollable">
+    <!-- <step-repositories></step-repositories> -->
+  </section>
+
+  <h3 class="module-settings-assistant-section-header-module-type"></h3>
+  <section id="module-list" class="module-assistant-step">
+    <!-- <step-modules></step-modules> -->
+  </section>
+
+  <h3 i18n="module-assistant.installation"></h3>
+  <section id="install" class="module-assistant-step scrollable">
+    <!-- <step-install></step-install> -->
+  </section>
+`;
+
+// const UPDATE_REPOSITORIES_INDEX = 0;
+const LANGUAGES_INDEX = 0;
+const REPOSITORIES_INDEX = LANGUAGES_INDEX + 1;
+const MODULES_INDEX = REPOSITORIES_INDEX + 1;
+const INSTALL_INDEX = MODULES_INDEX + 1;
+
+
+class AssistantStepsAdd extends HTMLElement {
```

Comment:
I would suggest to add the term "Module", so rename it to `AssistantStepsAddModule`. Let's make it crystal clear what this is all about.